src/CommunityToolkit.Maui.Core/Platform/StatusBar/StatusBar.net.cs

```diff
@@ -0,0 +1,8 @@
+namespace CommunityToolkit.Maui.Core.Platform;
+
+static partial class StatusBar
+{
+	static void PlatformSetColor(Color color) => throw new NotSupportedException($"{nameof(PlatformSetColor)} is only supported on net6.0-ios, net6.0-android, net6.0-maccatalyst and net6.0-windows");
```

Comment:
looks like the message is not correct. we do not support maccatalyst and windows

Reply:
Yup! Good catch! Fixed ✅